### PR TITLE
Click away preferences and fix focus profile

### PR DESCRIPTION
**Pull request by C0rn3j**
Fixes some of the issues in #1

### CODE REVIEW

File: webnovel/bot.py [hunk 1]
```diff
@@ -116,10 +116,13 @@ def signin(self, email, password):
         WebDriverWait(self.driver, self.timeout).until(
             EC.presence_of_element_located((By.CSS_SELECTOR, "a[title='My Profile']"))
         )
+        # click the preferences button away
+        signin_btn = self.driver.find_element_by_class_name('j_post_preference') # bt _m mw160 j_post_preference
```
Comment (m-haisham) [inline webnovel/bot.py:123]: preference popup seems to take sometime to load, so we should preferably wait and check for a time

```python
WebDriverWait(self.driver, 60).until(
    EC.presence_of_element_located((By.CSS_SELECTOR, ".j_post_preference"))
)
```

this throws a `TimeoutException` when timer has run out without finding the element. at which point we can just ignore the preference popup

___

the current button selection does not seem to work

```python
self.driver.find_element_by_css_selector('.j_post_preference button')
```

above code selects the first button that appear under preference panel

File: webnovel/bot.py [hunk 2]
```diff
@@ -504,7 +507,7 @@ def _focus_profile(self):
 
         :return: profile button element
         """
-        profile_button = self.driver.find_element_by_css_selector("div[class^='g_user'][class*='_hover']")
-        ActionChains(self.driver).move_to_element(profile_button).perform()
+        profile_button = self.driver.find_element_by_css_selector("div[class^='g_user'][class*='g_dropdown']")
+        ActionChains(self.driver).click(profile_button).perform()
```
Comment (m-haisham) [inline webnovel/bot.py]: this can be shortened to

```python
profile_button.click()
```

**Review by m-haisham — CHANGES_REQUESTED** [review]
Fix the suggested areas please

File: webnovel/bot.py (hunk 1 above)
Comment (C0rn3j) [reply webnovel/bot.py:123]: >the current button selection does not seem to work

It works for me. Your code doesn't.

I suspect A/B testing. Are we getting the same page code?

image

Comment (C0rn3j) [reply webnovel/bot.py:120]: EDIT: Actually it doesn't work for me either now, huh.

Comment (C0rn3j) [reply webnovel/bot.py:123]: I added the wait which indeed seemed to be an issue and used the shorter click function in the other place.

Comment (m-haisham) [reply webnovel/bot.py:123]: indeed, my bad. I mistyped. i meant.

```python
self.driver.find_element_by_css_selector('.j_preference_form button')
```

preference form doesnt appear all the time and without an exception catch some cases wont work.

But, this seems to work so i'll merge and add that later

**Pull request merged**